<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PE tab COPASI importer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblDir">
      <property name="text">
       <string>Benchmark dir: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txtDir"/>
      </item>
      <item>
       <widget class="QToolButton" name="cmdBrowseDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblOutDir">
      <property name="text">
       <string>Output dir:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="txtOutDir"/>
      </item>
      <item>
       <widget class="QToolButton" name="cmdBrwoseOutDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="lstModelDirs"/>
      <widget class="QTabWidget" name="wdgDetail">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabGeneral">
        <attribute name="title">
         <string>General</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QListWidget" name="lstModels"/>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="chkTransformData">
             <property name="statusTip">
              <string>If checked, measurement data is transformed in case 'log' or 'log10' is defined in the observableTransformation column</string>
             </property>
             <property name="text">
              <string>Transform Measurement Data</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdConvert">
             <property name="text">
              <string>Convert</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdOpenModelDir">
             <property name="text">
              <string>Open Model Dir</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cmdOpenInCOPASI">
             <property name="text">
              <string>COPASI</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="chkPlotProgressOfFit">
             <property name="statusTip">
              <string>If checked, Progress of Fit plot is added</string>
             </property>
             <property name="text">
              <string>Progress</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkPlotResult">
             <property name="statusTip">
              <string>If checked, parameter estimation result plot is added</string>
             </property>
             <property name="text">
              <string>Result</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkPlotResultPerExperiment">
             <property name="statusTip">
              <string>If checked, result plot per experiment is added</string>
             </property>
             <property name="text">
              <string>Result per Experiment</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkPlotResultPerDependent">
             <property name="statusTip">
              <string>If checked, result plot per dependent is added</string>
             </property>
             <property name="text">
              <string>Result per Dependent</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="chkWriteReport">
             <property name="statusTip">
              <string>If checked, a report file will be automatically written to NAME_report.txt</string>
             </property>
             <property name="text">
              <string>Write Report</string>
             </property>
            </widget>
           </item>
           </layout>
          </item>
         </layout>
       </widget>
       <widget class="QWidget" name="tabData">
        <attribute name="title">
         <string>Data</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="txtData"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdBrowseDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotBrowseBenchmarkDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdBrwoseOutDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotBrowseOutputDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstModelDirs</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotModelDirSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtDir</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSetBenchmarkDir(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtOutDir</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotSetOutputDir(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdConvert</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotConvert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdOpenInCOPASI</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotOpenInCOPASI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdOpenModelDir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotOpenModelDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstModels</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotModelSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Open</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotBrowseBenchmarkDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotBrowseBenchmarkDir()</slot>
  <slot>slotBrowseOutputDir()</slot>
  <slot>slotOpenInCOPASI()</slot>
  <slot>slotConvert()</slot>
  <slot>slotOpenModelDir()</slot>
  <slot>slotModelDirSelected()</slot>
  <slot>slotModelSelected()</slot>
  <slot>slotSetBenchmarkDir(QString)</slot>
  <slot>slotSetModelDir(QString)</slot>
  <slot>slotSetModel(QString)</slot>
  <slot>slotSetOutputDir(QString)</slot>
 </slots>
</ui>
